src/types/instruction.rs

```diff
@@ -78,3 +81,38 @@ impl Instruction {
         }
     }
 }
+
+// Returns True if the given instruction looks like a call instruction.
+pub(crate) fn is_call_instruction(encoded_instruction: &BigInt, imm: Option<BigInt>) -> bool {
```

Comment:
Why is `encoded_instruction` a `BigInt`?

Reply:
Maybe use an `Option<&BigInt>` for the immediate?

Reply:
`encoded_instruction` was left as a BigInt in order to reduce the complexity of `get_traceback_entries`. `is_call_instruction` is only used inside `get_traceback_entries`., which is on the longer side, so it was a bit more convenient to have the type conversion inside the simpler method `is_call_instruction`

Reply:
Added reference to `immediate `argument in `is_call_instruction` and `decode_instruction`